src/gsi/gridmod.F90

```diff
@@ -179,6 +179,7 @@ module gridmod
 
   logical wrf_nmm_regional  !
   logical fv3_regional      ! .t. to run with fv3 regional model
+  logical fv3_cmaq_regional ! .t. to run with fv3_cmaq_regional model
```

Comment:
@TingLei-NOAA does this satisfy your earlier comment or should it be more descriptive?

Reply:
@CoryMartin-NOAA Yes, this description is enough for this parameter . Thanks!